### DISCUSSION TITLE
Support for Tinkers' Construct Tables as Backpack Upgrades

### DISCUSSION THREAD

**Discussion by Darkosto** [P3pp3rF1y/SophisticatedBackpacks#520] [Feature / Idea Archive]
Hello! 

Would you consider adding a backpack upgrade that would allow you to either mimic Tinkers' Construct worktables or possibly just a repair station to allow Tool Repairs without needing to carry the Tinkers' Station or Anvil with you. In some cases, having Repair Kits wouldn't be an easy option.

Thanks!
Darkosto

**Comment by P3pp3rF1y** [comment]
Having tinker station in backpack sounds good to me for upgrades / repairs. I will just need to take a look at what exactly it means to add a compat for that.

**Reply by Darkosto** [reply]
Hey @P3pp3rF1y 

Have you had any luck on looking into this? Thanks so much and I've been loving the mod. Keep up the good work <3
